release_notes/release-notes.adoc

```diff
@@ -9,3 +9,4 @@ toc::[]
 {product-title} is released as an Operator that is supported on {platform}.
 
 include::modules/release-notes-1.0.adoc[leveloffset=+1]
+include::modules/release-notes-1.0.1.adoc[leveloffset=+1]
```

Comment:
I know this is out of scope for merge review these days, but I think most RNs put the newer version RNs ahead of the older versions (at least I do it that way for my operator RNs)

Reply:
Good call, thank you!